### DISCUSSION TITLE
support mobx/missing-observer option to set a path

### DISCUSSION THREAD

**Discussion by iMoses-Apiiro** [mobxjs/mobx#3236] [Ideas]
The mechanism for detecting what is a component today is too generic and can catch many false positives.
It'd be nice to be able to set a pattern/glob (or an array of) for matching files to avoid these false positives.

I know I can use an override but I think it's such a common use case it should be part of the rule itself.

**Comment by urugator** [comment]
> it's such a common use case

Therefore it's already part of eslint, so you don't have to implement (+ test + document + maintain) it yourself. I don't see a reason why there should be a second (non-standard) way of doing the same thing.
